### PR TITLE
sci-physics/hepmc: Add 3.2.6, add missing dependencies

### PR DESCRIPTION
**Pull request by APN-Pucky**
Fortran and numpy were missing as dependencies.

Closes: https://bugs.gentoo.org/905129
Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

File: sci-physics/hepmc/hepmc-3.2.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+DESCRIPTION="Event Record for Monte Carlo Generators"
+HOMEPAGE="https://hepmc.web.cern.ch/hepmc/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.cern.ch/hepmc/HepMC3"
+else
+	SRC_URI="https://hepmc.web.cern.ch/hepmc/releases/${MYP}.tar.gz"
+	S="${WORKDIR}/${MYP}"
+	KEYWORDS="~amd64"
```
Comment (AndrewAmmerlaan) [inline sci-physics/hepmc/hepmc-3.2.6.ebuild:21]: Any particular reason we are dropping the `x86` keyword?

**Review by AndrewAmmerlaan — APPROVED** [review]
LGTM, but why do we drop `x86`?

```
DeadUrl: version 2.06.09-r2: SRC_URI: connection failed: http://lcgapp.cern.ch/project/simu/HepMC/download/HepMC-2.06.09.tar.gz
```
I think we can drop this version, it is unfetchable upstream.

File: sci-physics/hepmc/hepmc-3.2.6.ebuild (hunk 1 above)
Comment (APN-Pucky) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild:21]: Oh, i think i did it again... I keep thinking I should only set the keywords for what I actually did/can test.

Comment (APN-Pucky) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild:21]: Wouldn't dropping that version need stabelization of the other `:2` version (first)?

Comment (AndrewAmmerlaan) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild:21]: > Wouldn't dropping that version need stabelization of the other `:2` version (first)?

Yes you are right. This package is long overdue for a stabilization request:
```
    StableRequest
    slot(2) no change in 248 days for unstable keywords: [ ~amd64, ~x86 ]
    StableRequest
    slot(3) no change in 65 days for unstable keywords: [ ~amd64, ~x86 ] 
```

Could you open one? Maybe we can also request stabilization for slot 3.

Comment (AndrewAmmerlaan) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild:21]: > Oh, i think i did it again... I keep thinking I should only set the keywords for what I actually did/can test.

For major version bumps with big changes this is often a good idea. For regular version bumps it is okay to just keep the keywords that are already there. By the way, for `x86` it is also possible to test in a container on `amd64`.

Comment (APN-Pucky) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild]: I would only consider 3.2.6 good enough for stabilization in slot 3 (cf. https://gitlab.cern.ch/hepmc/HepMC3/-/issues/59)

Comment (AndrewAmmerlaan) [reply sci-physics/hepmc/hepmc-3.2.6.ebuild]: > I would only consider 3.2.6 good enough for stabilization in slot 3 and that's new (cf. https://gitlab.cern.ch/hepmc/HepMC3/-/issues/59)

Alright, then lets just stabilize only slot 2 and revisit stabilization of slot 3 later.

**Comment by AndrewAmmerlaan** [comment]
Nice work, Thanks :+1:

**Pull request closed without merging**